src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  /* justify-content: center;
+  align-items: center; */
+}
+
+.no_active {
+  background-image: url("images/star.svg");
+}
```

Comment:
if star isn't active it's just a star, so you can add those styles to single star, you don't need to add another class